sphinx_airflow_theme/README.md

```diff
@@ -55,6 +55,29 @@ pip install 'https://github.com/apache/airflow-site/releases/download/v0.0.1/sph
 Python packages for your PRs is available as downloadable artifact in GitHub Actions after
 the CI builds your PR.
 
+# Configuration
+
+A theme that supports the following configuration options:
+
+## `sphinx_airflow_theme_navbar_links`
+
+The list of links that should be available in the navigation bar at the top of the pages.
+
+**Example values:**
+```
+[
+    {'href': '/docs/', 'text': 'Documentation'}
+]
+```
```

Comment:
Does the links will be rendered in provided order? If yes it would be nice to mention it

Reply:
Added